fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -121,6 +151,127 @@ private fun AdjustedProductRatingCard(
     }
 }
 
+@Composable
+private fun HighlightsCard(
+    highlights: Map<HighlightType, List<String>>,
+    highlightsForCompactMode: Map<HighlightType, List<String>>,
```

Comment:
I wonder if instead of passing the 2 maps as params we just pass the original `highlights` map and have some kind of `getCompactHighlights` fun we can call from here on `highlights` - this would mean we're working from a single source of truth within this func

Reply:
✅